sdks/java/extensions/zetasketch/src/main/java/org/apache/beam/sdk/extensions/zetasketch/HllCountMergePartialFn.java

```diff
@@ -54,10 +54,16 @@ private HllCountMergePartialFn() {}
     return null;
   }
 
+  @Nullable
   @Override
   public HyperLogLogPlusPlus<HllT> addInput(
       @Nullable HyperLogLogPlusPlus<HllT> accumulator, byte[] input) {
-    if (accumulator == null) {
+    if (input == null) {
+      throw new NullPointerException(
```

Comment:
Consider using `checkArgument`?

Reply:
Nice catch! (But this is outdated since I changed the behavior.)